test/local/rns.available.test.ts

```diff
@@ -21,14 +22,18 @@ import { Options } from '../../src/types';
 import { labelhash } from '../../src/utils';
 import { ZERO_ADDRESS } from '../../src/constants';
 
+const web3Instance = web3 as unknown as Web3;
+const rsk3 = new Rsk3(web3.currentProvider);
```

Comment:
```suggestion
const rsk3Instance = new Rsk3(web3.currentProvider);
```

for consistency with `web3Instance`